en/04_Optional_Features/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Optional features
+---
+
+# Optional features
+
+Silverstripe CMS can be highly customised using both supported and community-made modules. You can find these [one Packagist](https://packagist.org/?type=silverstripe-vendormodule).
+
+This section highlights some of the CMS features provided by the [Silverstripe CMS supported modules](/project_governance/supported_modules/).
+In each case you'll need to ensure you have the module installed in your project in order to use the features described here.
```

Comment:
Should either have a newline above this or have both sentences on one line

Reply:
We explicitly don't have a standard for whether to break up lines or not, it doesn't affect anything.